docs/pages/includes/database-access/guides.mdx

```diff
@@ -7,6 +7,7 @@
 - [GCP Cloud SQL PostgreSQL](../../database-access/guides/postgres-cloudsql.mdx): Connect GCP Cloud SQL PostgreSQL database.
 - [GCP Cloud SQL MySQL](../../database-access/guides/mysql-cloudsql.mdx): Connect GCP Cloud SQL MySQL database.
 - [Azure PostgreSQL & MySQL](../../database-access/guides/azure-postgres-mysql.mdx): Connect Azure PostgreSQL or MySQL.
+- [Active Directory Azure SQL Server (Preview)](../../database-access/guides/azure-sql-server-ad.mdx): Connect Azure SQL Server with Active Directory authentication.
```

Comment:
```suggestion
- [Azure SQL Server (Preview)](../../database-access/guides/azure-sql-server-ad.mdx): Connect Azure SQL Server with Azure Active Directory authentication.
```
2 nits:
* start the link name with the name of the cloud provider like the other links do for consistency.
* Clarify that "Active Directory" is referring to "Azure Active Directory" authentication, which is different. In my opinion the term "Active Directory" is overloaded and I blame Microsoft for that 😄